main.go

```diff
@@ -265,6 +269,13 @@ func main() {
 	}
 
 	if autoScalingRunnerSetOnly {
+		var newWorkqueueRateLimiter func() workqueue.RateLimiter
+		if disableWorkqueueBucketRateLimiter {
+			newWorkqueueRateLimiter = workqueue.DefaultItemBasedRateLimiter
```

Comment:
Not sure what version of workqueue this code is on, but in latest versions `DefaultItemBasedRateLimiter` is considered deprecated: "Deprecated: DefaultItemBasedRateLimiter is deprecated, use DefaultTypedItemBasedRateLimiter instead."